index.bs

```diff
@@ -390,6 +390,74 @@ by a custom configuration.
 {}
    ```
 
+# Security Considerations {#security-considerations}
+
+The Sanitizer API is intended to prevent DOM-Based Cross-Site Scripting
+by traversing a DocumentFragment and removing elements and attributes
+according to a configuration. The specified API must not support
+the construction of a Sanitizer object that leaves script-capable markup in
+and doing so would be a bug in the threat model.
+
+That being said, there are security issues which the correct usage of the
+Sanitizer API will not be able to protect against and the scenarios will be
+laid out in the following sections.
+
+## Server-Side Reflected and Stored XSS {#server-side-xss}
+
+<em>This section is not normative.</em>
+
+The Sanitizer API operates solely in the DOM and adds a capability to traverse
+and filter an existing DocumentFragment. The Sanitizer does not address
+server-side reflected or stored XSS.
+
+## DOM clobbering {#dom-clobbering}
+
+<em>This section is not normative.</em>
+
+DOM clobbering describes an attack in which malicious HTML confuses an
+application by naming elements through `id` or `name` attributes such that
+properties like `children` of an HTML element in the DOM are overshadowed by
+the malicious content.
+
+The Sanitizer API does not protect DOM clobbering attacks in its
+default state, but can be configured to remove `id` and `name` attributes
+through a configuration.
+
+## XSS with Script gadgets {#script-gadgets}
```

Comment:
If you wish to link to something, we usually use https://github.com/google/security-research-pocs/tree/master/script-gadgets when describing script gadgets. https://research.google/pubs/pub46450/ would work too.

Reply:
Thanks!